clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/ThreadPoolServiceConfigurator.java

```diff
@@ -61,7 +61,9 @@ public ServiceConfigurator configure(OperationContext context, ModelNode model)
         long keepAliveTime = this.definition.getKeepAliveTime().resolveModelAttribute(context, model).asLong();
         boolean nonBlocking = this.definition.isNonBlocking();
         if (this.definition == ThreadPoolResourceDefinition.NON_BLOCKING) {
-            maxThreads *= ProcessorInfo.availableProcessors();
+            int availableProcessors = ProcessorInfo.availableProcessors();
+            minThreads *= availableProcessors;
+            maxThreads *= availableProcessors;
```

Comment:
The read-resource-description text description of these 2 attributes should mention that this multiplication occurs. (I think continuing to share the text across pool types is ok; the text can mention that the behavior is only for a non-blocking pool.)

I don't think this point should block this change getting in for WF 21 if that is wanted; it can come later.

Reply:
That's valid.  I've opened https://issues.redhat.com/browse/WFLY-13927 to track this.